### PR TITLE
bk 0.5.3 (new formula)

### PR DESCRIPTION
**Pull request by reishoku**
Signed-off-by: KOSHIKAWA Kenichi <reishoku.misc ~~@~~ pm.me>

-----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/bk.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  homepage "https://github.com/aeosynth/bk"
+  url "https://github.com/aeosynth/bk/archive/refs/tags/v0.5.3.tar.gz"
+  sha256 "c9c54fa2cd60f3ca0576cab8bdd95b74da4d80c109eb91b5426e7ee0575b54f1"
+  license "MIT"
+  head "https://github.com/aeosynth/bk.git", branch: "master"
+
+  livecheck do
+    url :stable
+  end
+
```
Comment (chenrui333) [inline Formula/bk.rb:12]:
```suggestion
```

Comment (reishoku) [reply Formula/bk.rb]: Thanks for the comment. Fixed.

File: Formula/bk.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    output = shell_output "#{bin}/bk", 1
+    assert_match output, "init\ error:\ no\ path\ arg\ and\ no\ valid\ save\ file\n"
```
Comment (chenrui333) [inline Formula/bk.rb:17]: we have a minimal `test.epub`, so let's use that
```suggestion
    test_epub = test_fixtures("test.epub")
    output = pipe_output("#{bin}/bk --meta #{test_epub}")
    assert_match "language: en", output
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @reishoku!

**Pull request closed without merging**